src/python/foglamp/core/service_registry/service_registry.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Core server module"""
+
+import time
+
+from aiohttp import web
+
+from foglamp.core.service_registry.instance import Service
+
+__author__ = "Amarendra Kumar Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+__start_time = time.time()
+
+
+async def ping(request):
+    since_started = time.time() - __start_time
+
+    return web.json_response({'uptime': since_started})
+
+async def register(request):
+    """
+    Register a service
```

Comment:
please tell required params; check with @MarkRiddoch if adding protocol will make sense?

Reply:
Yes, it would make sense to add the protocol because in future it may be HTTP or HTTPS